TilerServer/TilerWebModule.pas

```diff
@@ -3747,6 +3748,33 @@ function TSliceDiffGeometryPolygonLR.GenerateTileCalc(const aExtent: TExtent; aB
   else Log.WriteLn('TSliceDiffGeometryPolygonLR layer '+fLayer.LayerID.ToString+': no palette defined', llError);
 end;
 
+procedure TSliceDiffGeometryPolygonLR.UpdateBufferExtent(const aExtent: TExtent; capacityFactor, aPixelWidth, aPixelHeight: Double; var bufferExtent: TExtent);
+var
+  maxValue: Double;
+  maxPixels: Double;
+  isgop: TPair<TWDID, TSliceGeometryPolygonLRObject>;
+begin
+  maxValue := double.NaN;
+  for isgop in (fCurrentSlice as TSliceGeometryPolygonLR).fGeometries do
+  begin
+    if not isgop.Value.value.IsNan then

```

Comment:
do the abs on max value here so negative values are used for raging too.
The abs on the end can then be omitted..